src/servers/src/http.rs

```diff
@@ -599,6 +599,7 @@ impl HttpServer {
     fn route_influxdb<S>(&self, influxdb_handler: InfluxdbLineProtocolHandlerRef) -> Router<S> {
         Router::new()
             .route("/write", routing::post(influxdb_write))
+            .route("/api/v2/write", routing::post(influxdb_write))
```

Comment:
We have `/write` and `/api/v2/write`, they seem not consistent with each other. Is it able to change one? Like, do we have to use `/api/v2/write` to adapt influxdb's official sdk?

Reply:
Influxdb go sdk v2 will add "/api/v2" after the url.
https://github.com/influxdata/influxdb-client-go/blob/114e28a215dfe0d9e0763e822c9b2e961b1f35fa/api/http/service.go#L62C1-L79C2

So we need two APIs. `/write` and `/api/v2/write`.

And when influxdb sdk builds the `client`, the required `addr` parameter can also be consistent.
https://github.com/Fengys123/influxdb_write_example/blob/main/main.go#L21
https://github.com/Fengys123/influxdb_write_example/blob/main/main.go#L52